.gitlab-ci.yml

```diff
@@ -699,6 +699,11 @@ sonarqube_cov_:
     - .quick_test_short_lived_condition
     - .before_script_template
     - .use_gko-cuda101-openmpi-gnu8-llvm7-intel2019
+  tags:
+    - private_ci
+    - controller
+    - cpu
+    - nla-gpu
```

Comment:
any reason for the requirment nla-gpu?

Reply:
Yes, see the commit message. It fails on HoreKa all the time, with a nodejs error. As we were submitting a job to run this, this could be network-related for example.